### DISCUSSION TITLE
有clash有不兼容的情况吗？uwp lookback 已经设置了

### DISCUSSION THREAD

**Discussion by QAQ-Huaxia** [ywmoyue/biliuwp-lite#252] [General]
uwp lookback 已经设置且保存了，但大部分情况下开启clash就出现biliuwp-lite"断网"的情况（小概率不出现），clash是系统代理或者是TUN模式都可能出现这种情况，且clash我已经添加规则和uwp lookback设置且保存好了，不知道啥原因。
- 'DOMAIN-KEYWORD,bilibi,DIRECT'
- PROCESS-NAME,BiliLite.exe,DIRECT
[20230808.log](https://github.com/ywmoyue/biliuwp-lite/files/12293209/20230808.log)
![uwp lookback配置](https://github.com/ywmoyue/biliuwp-lite/assets/67990248/8a0ec130-2aea-4b12-80e3-e7628738adc2)

**Comment by abandonstudy** [comment]
clash   Tun mode，我就算没解锁uwp应用回环，哔哩哔哩 UWP也能正常使用网络，有点奇怪，有没有那个√，都不影响使用
![Uploading image.png…]()

**Comment by abandonstudy** [comment]
![image](https://github.com/ywmoyue/biliuwp-lite/assets/94209629/f972015e-6cb6-4d4e-b707-fcc781e0e805)

**Comment by QAQ-Huaxia** [comment]
啊？这么奇怪的吗？算了，也不太影响使用，有时候能用有时候不能就觉得有点奇怪，过来问问

**Comment by ywmoyue** [comment]
我不常用clash，给不了什么帮助
